src/app/customizer/gradient-generator/page.js

```diff
@@ -0,0 +1,302 @@
+"use client";
+import Nav from "@/app/assets/nav";
```

Comment:
Replace it with the search component
```suggestion
import Search from "@/app/assets/search";
```